README.md

```diff
@@ -154,7 +155,15 @@ frontend --> checkoutservice
 frontend --> currencyservice
 frontend --> recommendationservice --> productcatalogservice
 frontend --> shippingservice
+frontend --> |evalFlag| featureflagfeservice
 
+productcatalogservice --> |evalFlag| featureflagfeservice
+
+featureflagbeservice(Flag Server):::erlang
+featureflagfeservice(Flag UI/API):::erlang
+featureflagstore[(Flag Store<br/>&#40Blob/DB&#41)]
```

Comment:
Do we want to callout the exact technology (Postgres) since we do with Redis just to stay consistent?

Reply:
Yep good call out